dec_msgs/Object object
byte operation
string attached_to

# Add or update
byte ADD=0

# Remove
byte REMOVE=1

# Attach to a frame
byte ATTACH=2

# Detach from a frame
byte DETACH=3
